modules/formulize/admin/formindex.php

```diff
@@ -826,7 +826,7 @@ function patch40() {
 											break;
```

Comment:
I genuinely don't know why these changes are showing up. I copied the file from `master` and put it in my branch to remove any conflicts yet these changes are still showing up.